roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -78,8 +80,10 @@ def get_required_ovs_version(self):
         Else ensure Open vSwitch version 2.4"""
         openshift_version = self.get_openshift_version()
 
-        if float(openshift_version) < 3.5:
+        if _version_lt(openshift_version, "3.5"):
             return self.openshift_to_ovs_version["3.4"]
+        if _version_lt("3.7", openshift_version):
+            return self.openshift_to_ovs_version["3.7"]
```

Comment:
Suggestion: do not hard code "3.7". If the version is greater than the latest version we know about, we can treat it as the latest known. If the version is lower than the earliest version we know about, treat it as the earlier known (i.e., do not hard code the "3.4" either.

That way, whenever a new version comes in, all we have to do is to add 1 line -- or perhaps none, if the version of the deps stays the same, meaning the checks should likely be working on day one after starting a new release cycle.